rpcs3/Emu/RSX/VK/VKHelpers.h

```diff
@@ -2256,7 +2257,7 @@ namespace vk
 			m_instance = nullptr;
 
 			//Check that some critical entry-points have been loaded into memory indicating presence of a loader
-			loader_exists = (vkCreateInstance != nullptr);
+			loader_exists = (&vkCreateInstance != nullptr);
```

Comment:
Not sure about this one on windows without vulkan RT installed. If the method is just a function description then it can be null. If its a dllimport, it cannot be nullptr. More research is needed.

Reply:
I didn't clarify - leave it in until I've had time to check the ABI

Reply:
Remove this variable as it is no longer useful I think. The application will not start without a vulkan runtime even on old hardware that does not support it anyway.